app/src/main/java/io/homeassistant/companion/android/settings/developer/DeveloperSettingsFragment.kt

```diff
@@ -93,14 +110,24 @@ class DeveloperSettingsFragment : DeveloperSettingsView, PreferenceFragmentCompa
     }
 
     override fun onThreadDebugResult(result: String, success: Boolean?) {
-        threadDebugDialog?.hide()
+        activeTaskDialog?.hide()
         AlertDialog.Builder(requireContext())
             .setTitle(commonR.string.thread_debug)
             .setMessage("${if (success == true) "✅" else if (success == null) "⚠️" else "⛔"}\n\n$result")
             .setPositiveButton(commonR.string.ok, null)
             .show()
     }
 
+    override fun onWebViewClearCacheResult(success: Boolean) {
+        lifecycleScope.launch {
+            delay(750L) // Add a delay to prevent the dialog 'flashing' if clearing completed quickly
```

Comment:
The flashing occur only if the cache is small enough, but if the cache is big we are adding 750ms on top of it. It's not a big deal but we could avoid this with a flow and a throttle operator. We can keep it like this since it's a debug screen it's not a big deal.

Reply:
That was my thought: this is a lot quicker to setup, keeps it simple and won't bother the user that much if the cache takes for example 30 seconds to clear.